test/TestCaseReader.cpp

```diff
@@ -169,6 +163,11 @@ string TestCaseReader::parseSimpleExpectations(istream& _file)
 		else if (line == "//")
 			result += "\n";
 		else
-			BOOST_THROW_EXCEPTION(runtime_error("Test expectations must start with \"// \"."));
+			fail("Test expectations must start with \"// \".");
 	return result;
 }
+
+void TestCaseReader::fail(std::string const& _message)
+{
+	BOOST_THROW_EXCEPTION(runtime_error(_message));
```

Comment:
The main purpose of BOOST_THROW_EXCEPTION is that it encodes the current line number into the exception, so you should not put it in a utility function.

Reply:
Ah, good point. I'll restore it then.

Reply:
Done.